sql/core/src/main/scala/org/apache/spark/sql/execution/joins/HashedRelation.scala

```diff
@@ -163,6 +164,55 @@ private[joins] object HashedRelation {
       new GeneralHashedRelation(hashTable)
     }
   }
+
+  def createLocalHashedRelation(
```

Comment:
Need to use a new method name because the default parameter has conflicts with overloading.